docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -72,6 +72,22 @@ The following table lists the available GX Cloud Expectations.
 
 6. Click **Save**.
 
+## View Expectation history
+
+View the Expectation history to determine when an Expectation was changed and who made the change. 
+
+1. In GX Cloud, click **Data Assets**.
+
+2. In the **Data Assets** list, click the Data Asset name.
```

Comment:
I find it a little strange that we're telling them to access this through the Data Assets page rather than the Expectation Suites page -- do we have a preferred route for people to access their Expectations?

Reply:
@Rachel-Reverie I've updated the procedure to use the Expectation Suites path.